code/stable/glassbr/src/cpp/InputParameters.cpp

```diff
@@ -238,29 +238,29 @@ void InputParameters::input_constraints() {
         std::cout << "." << std::endl;
         throw("InputError");
     }
-    set<string> set_glassTy = {"AN", "FT", "HS"};
-    if (!(set_glassTy.find(this->g) != set_glassTy.end())) {
+    set<string> set_g = {"AN", "FT", "HS"};
```

Comment:
wait, this diff seems to go in the opposite direction than I expected? `glassTy` is nicer than `g`? 

Now, it may well be that `glassTy` was the UID and `g` is the symbol name - is that so?

Reply:
Yes that is the case

Reply:
This feels like a regression to me. *However* you are doing what is being asked of you in this PR, so it's not "your fault", as it were.

So: I'll let this PR in as is, but could you please open an issue where we discuss these names? We have different ways of associating names to chunks, and we might want to use a different one here (UID was definitely wrong, `symbol` is better for sure, but maybe `shortname` would be better still, but that is not in-scope for this PR).

Reply:
I opened an issue. I think it would be best to use `shortname`. I also don't think it would take very long to do.